package.json

```diff
@@ -112,21 +113,20 @@
     "prop-types": "15.5.10",
     "raven-js": "3.15.0",
     "raw-loader": "0.5.1",
-    "react": "15.1.0",
-    "react-addons-css-transition-group": "15.1.0",
-    "react-addons-test-utils": "15.1.0",
-    "react-document-title": "2.0.1",
-    "react-dom": "15.1.0",
+    "react": "15.5.4",
+    "react-addons-css-transition-group": "15.5.2",
+    "react-addons-test-utils": "15.5.1",
+    "react-document-title": "2.0.3",
+    "react-dom": "15.5.4",
     "react-hot-loader": "1.3.0",
     "react-imageloader": "2.1.0",
     "react-onclickoutside": "5.3.1",
-    "react-redux": "4.4.5",
-    "react-spinkit": "1.1.4",
-    "react-stripe-checkout": "1.7.2",
-    "redux": "3.3.1",
-    "redux-batched-actions": "0.1.5",
-    "redux-mock-store": "1.0.2",
-    "redux-thunk": "2.1.0",
+    "react-redux": "5.0.5",
+    "react-spinkit": "3.0.0",
+    "redux": "3.6.0",
+    "redux-batched-actions": "0.2.0",
+    "redux-mock-store": "1.2.3",
+    "redux-thunk": "2.2.0",
     "semver": "4.3.6",
     "sentry-cli-binary": "1.13.3",
     "sinon": "2.0.0-pre",
```

Comment:
If you update this to 2.1 you can get rid of `sinon-as-promised` and use fancy new stuff like `callsFake`